net-p2p/kubo/kubo-0.30.0.ebuild

```diff
@@ -5,9 +5,17 @@ EAPI=8
 
 inherit go-module shell-completion systemd
 
+FS_MIG_V="1.0.1"
+FS_MIG_N="fs-repo-15-to-16"
+MY_FS_MIG="${FS_MIG_N}-v${FS_MIG_V}"
+FS_MIG_DIR="fs-repo-migrations-${MY_FS_MIG}"
```

Comment:
Would love a comment above these explaining why these vars are necessary and where they come from. Just for when someone else takes a look at the ebuild it's obvious.